src/decorators/withResources/addPagingResponse.js

```diff
@@ -0,0 +1,44 @@
+function transformMeta(meta) {
+  const { total, per_page, current_page, last_page, from, to } = meta
+
+  const api = {
+    hasNext: () => current_page < last_page,
+    hasPrevious: () => current_page !== 1,
```

Comment:
Q: Why are these methods, and not statically evaluated already?

Reply:
Same answer as the other one; It's because the docs stated this as a function. I'm not sure what's the gain in both